content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-sample.md

```diff
@@ -0,0 +1,641 @@
+---
+path: 'stream-developer-guides/streams/standalone-stream-sample/'
+title: 'Stream Application Development'
+description: 'Create your own microservices for Stream processing and deploy them manually'
+---
+
+In this guide, we develop three Spring Boot applications that use Spring Cloud Stream and deploy them to Cloud Foundry, to Kubernetes, and to your local machine.
+In another guide, we [deploy these applications by using Data Flow](%currentPath%/stream-developer-guides/streams/data-flow-stream/).
+By deploying the applications manually, you get a better understanding of the steps that Data Flow automates for you.
+
+The sections that follow describe how to build these applications from scratch.
+
+If you prefer, you can download a zip file containing the sources for these applications, unzip it, build it, and proceed to the [deployment](#deployment) section.
+
+You can [download the project](https://github.com/spring-cloud/spring-cloud-dataflow-samples/blob/master/dataflow-website/stream-developer-guides/streams/standalone-stream-sample/dist/usage-cost-stream-sample.zip?raw=true) that contains all three applications from your browser. You can also use the command line, as the following example shows:
+
+```bash
+wget https://github.com/spring-cloud/spring-cloud-dataflow-samples/blob/master/dataflow-website/stream-developer-guides/streams/standalone-stream-sample/dist/usage-cost-stream-sample.zip?raw=true -O usage-cost-stream-sample.zip
+```
+
+# Building the downloaded sample
+
+The stream apps may may be configured to run with a Kafka broker or RabbitMQ, using a common code base. The only difference is in the executable jar files. In order for them to work with a Kafka broker, they require a Kafka binder dependency (enabled by default). For RabbitMQ, they require the Rabbit binder.
+
+<!--TIP-->
+
+Additional Spring Cloud Stream binder implementations include Kafka Streams, Amazon Kinesis, Google PubSub (partner maintained), Solace PubSub+ (partner maintained), Azure Event Hubs (partner maintained). The binder selection is a build time step. The sample project uses a Maven profile to enable the corresponding binder.
+
+<!--END_TIP-->
+
+<!--TABS-->
+
+<!--Kafka-->
+
+To build the sample stream app for Kafka, from the root project directory:
```

Comment:
Maybe we can specify the directory name here?

Reply:
I see it is the directory from the unzipped archive and guess it is ok not to specify the name